cafebabel/templates/base.html

```diff
@@ -3,6 +3,21 @@
 <meta name=viewport content="width=device-width, minimum-scale=1, initial-scale=1, shrink-to-fit=no">
 <title>{% block meta_title %}pan-European news, features and opinions{% endblock %} – Cafebabel</title>
 <meta name=description content="{% block meta_description %}{% endblock %}">
+<!--Social-->
+<meta property="og:url" content="{{ request.url }}">
+<meta property="og:title" content="{{ self.meta_title() }}">
+<meta property="og:image" content="{% block meta_image_url %}{% endblock %}"> <!-- to discuss -->
```

Comment:
What is the `<!-- to discuss -->` for?
If it's a comment for development, please use `{# to discuss #}` instead so it does not go into the response HTML